### DISCUSSION TITLE
build amplication-cli failed

### DISCUSSION THREAD

**Discussion by mahtafreshi** [amplication/amplication#1278] [Q&A] [answered]
I am following installation instructions from scratch. when running npm run bootstrap, I am getting the following error :

src/configured-command.ts:50:7 - error TS2322: Type '(input: import("C://My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").RequestInfo, init?: import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").RequestInit | undefined) => Promise<...>' is not assignable to type '(input: RequestInfo, init?: RequestInit | undefined) => Promise<Response>'.
  Types of parameters 'input' and 'input' are incompatible.
    Type 'RequestInfo' is not assignable to type 'import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").RequestInfo'.
      Type 'Request' is not assignable to type 'RequestInfo'.
        Type 'Request' is not assignable to type 'import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").Request'.
          Types of property 'body' are incompatible.
            Type 'ReadableStream<Uint8Array> | null' is not assignable to type 'import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").ReadableStream<Uint8Array> | null'.
              Type 'ReadableStream<Uint8Array>' is not assignable to type 'import("C:/My Projects/amplication-master/packages/amplication-cli/node_modules/cross-fetch/lib.fetch").ReadableStream<Uint8Array>'.
                Types of property 'getReader' are incompatible.
                  Type '{ (options: { mode: "byob"; }): ReadableStreamBYOBReader; (): ReadableStreamDefaultReader<Uint8Array>; }' is not assignable to type '() => ReadableStreamDefaultReader<Uint8Array>'.

50       fetch,

**Comment by yuval-hazaz** [comment]
I am trying to reproduce with no success. 
I also deleted the node_modules on amplication-cli and npm run bootstrap is working fine. 
I will try to reproduce on a fresh install and keep you updated

**Comment by yuval-hazaz** [comment]
can you also please try to run 
```
npm i
```
 in amplication-cli folder?

**Reply by mahtafreshi** [reply]
I cloned the project and ran "npm i" and "npm install" on the root of project, getting same error. the issue cause when I run "npm run bootstrap"

**Reply by mahtafreshi** [reply]
![image](https://user-images.githubusercontent.com/51502712/111258560-5925ad80-8671-11eb-89e1-a12d6390df8a.png)

**Comment by yuval-hazaz** [comment]
Join our discord channel, I will try to help you there
https://discord.gg/b8MrjU6

**Reply by mahtafreshi** [reply]
I have added it to the support channel

**Comment by yuval-hazaz** [comment, marked as answer]
It turned out that "cross-fetch" released a new version 3.1.0 three days ago and it breaks the build... 
I am still not sure why it breaks,  as there should be no breaking changes between 3.0.6 and 3.1.0. 
I will inves||tigate, and fix our side if needed - otherwise, I will change the package.json to use "3.0.6" instead of "^3.0.6" - until @graphql-tools will update their dependencies
